src/hotspot/share/runtime/threadSMR.cpp

```diff
@@ -76,7 +76,7 @@ volatile uint         ThreadsSMRSupport::_deleted_thread_time_max = 0;
 volatile uint         ThreadsSMRSupport::_deleted_thread_times = 0;
 
 // The bootstrap list is empty and cannot be freed.
-ThreadsList ThreadsSMRSupport::_bootstrap_list = ThreadsList(0);
+ThreadsList ThreadsSMRSupport::_bootstrap_list{0};
```

Comment:
To anyone unfamiliar with the details of direct initialization this is very obscure. I don't know how to interpret this given a ThreadsList has four member fields. Wouldn't `{}` have the same effect of implicitly zero-initializing all the fields?
Stylistically I find this syntax jarring, an equals sign in there would look far more natural to me. :(